assignment10/controllers/employeeController.js

```diff
@@ -0,0 +1,83 @@
+var mongoose = require('mongoose'), 
+  employees = mongoose.model('employees')
+  project = mongoose.model('projects');
+
+exports.allEmployees = function(req, res){
+  employees.find({}, {name : 1,username : 1,_id : 0},(err, result) => {
```

Comment:
@abansal26 Please follow the same pattern, If you're using arrow operator then use arrow operator in all places.

Reply:
Changed to function(). Commit ID - 0836b72